internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func (ns *NodeServer) populateVolOptions(
+	ctx context.Context,
+	req *csi.NodeStageVolumeRequest,
+	disableInUseChecks bool,
+	isStaticVol bool,
```

Comment:
the whole request is passed, is it not possible to detect `isStaticVol` from that?

Reply:
Can be done,  and its done :)